src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLoggerOptions.cs

```diff
@@ -123,6 +123,13 @@ public TimeSpan QuickPulseInitializationDelay
         /// </summary>
         public HttpAutoCollectionOptions HttpAutoCollectionOptions { get; set; } = new HttpAutoCollectionOptions();
 
+        /// <summary>
+        /// Gets or sets a value that removes the query parameters from functions URLs. True by default.
+        /// When true, the query parameters are stripped off of the functions URL for logging HTTP trigger calls.
+        /// When false, query parameters are logged.
+        /// </summary>
+        public bool SanitizeRequestURL { get; set; } = true;
```

Comment:
I don't love this name... can it be something specific about the QueryString? Like `EnableQueryStringTracing` or something? (that's not great either... but `Sanitize` makes me think we're doing something to obscure the whole url rather than just touching the QueryString)

Reply:
Agreed, having the option be a new opt-in feature rather than a new opt-out seems prudent. Renamed to EnableQueryStringTracing for lack of a better descriptor, set the default value to true, and flipped all the logic